spec/support/factories.rb

```diff
@@ -15,6 +15,18 @@
     author_id '1'
   end
 
+  factory :dynamic_book, class: Book do
+    sequence(:title) do |i|
+      i = i.to_s.rjust(2, "0") if i < 10
```

Comment:
Yes, Ordering by title when querying without prepended zero results in single digit book titles to be mixed in instead of at the beginning of the list.